.travis.yml

```diff
@@ -12,7 +12,7 @@ install:
 script:
   # This unencryptes envvars.bash.enc into envvars.bash so that we can run the tests
   - openssl aes-256-cbc -K $encrypted_9c9444fdc7c6_key -iv $encrypted_9c9444fdc7c6_iv -in envvars.bash.enc -out envvars.bash -d
-  - ./testandcover.bash
+  - make docker-genassets test-and-cover
```

Comment:
The docker-genassets will not be required after we merge https://github.com/getlantern/flashlight-build/pull/178